droidlet/memory/memory_filters.py

```diff
@@ -310,13 +310,14 @@ def handle_where(self, agent_memory, where_clause, memtype):
                     raise Exception("error in subquery {}".format(where_clause))
 
         triples = agent_memory.get_triples(**where_clause)
-        node_children = agent_memory.node_children[memtype]
-        if len(triples) > 0:
-            return [
-                t[0] for t in triples if agent_memory.get_node_from_memid(t[0]) in node_children
-            ]
+        if where_clause.get("subj"):
+            memids = [t[2] for t in triples]
```

Comment:
hm maybe I am not understanding this right, but `"subj"` exists then memids are in t[0], no ?